kingpin/actors/group.py

```diff
@@ -71,6 +73,12 @@ def __init__(self, *args, **kwargs):
         """
         super(BaseGroupActor, self).__init__(*args, **kwargs)
 
+        if type(self.option('contexts')) == dict:
+            try:
+                open(self.option('contexts')['file'])
+            except IOError as e:
```

Comment:
If someone passes in a `dict` that doesn't have the `file` key, this will throw a different exception. Test for that. :)